### PR TITLE
KUZ-378 OMG Who Am I? (identity crisis)

### PR DESCRIPTION
**Pull request by xbill82**

### CODE REVIEW

**Comment by j33f** [comment]
:+1:

File: dist/kuzzle.js [hunk 1]
```diff
@@ -754,6 +754,23 @@ Kuzzle.prototype.checkToken = function (token, callback) {
...
+ * Fetches the current user.
+ *
+ * @param  {function} callback  The callback to be called when the response is
+ *                              available. The signature is `function(error, response)`.
+ * @return {Kuzzle}             The Kuzzle instance to enable chaining.
+ */
+Kuzzle.prototype.whoAmI = function (callback) {
+  var self = this;
+
+  this.callbackRequired('Kuzzle.checkToken', callback);
```
Comment (AnthonySendra) [inline dist/kuzzle.js]: I think it's must be `this.callbackRequired('Kuzzle.whoAmI', callback);`

File: dist/kuzzle.min.map [hunk 2]
```diff
@@ -1 +1,5 @@
...
\ No newline at end of file
+<<<<<<< HEAD
```
Comment (AnthonySendra) [inline dist/kuzzle.min.map]: Did you just commit a merge conflict ? :D

File: src/kuzzle.js [hunk 3]
```diff
@@ -479,6 +479,23 @@ Kuzzle.prototype.checkToken = function (token, callback) {
...
+ * Fetches the current user.
+ *
+ * @param  {function} callback  The callback to be called when the response is
+ *                              available. The signature is `function(error, response)`.
+ * @return {Kuzzle}             The Kuzzle instance to enable chaining.
+ */
+Kuzzle.prototype.whoAmI = function (callback) {
+  var self = this;
+
+  this.callbackRequired('Kuzzle.checkToken', callback);
```
Comment (AnthonySendra) [inline src/kuzzle.js]: I think it's must be `this.callbackRequired('Kuzzle.whoAmI', callback);`

File: dist/kuzzle.min.map (hunk 2 above)
Comment (xbill82) [inline dist/kuzzle.min.map]: Yes, it will be fixed by Travis build on merge to master.

**Comment by scottinet** [comment]
+1

**Pull request merged**